test/DotNetWorkerTests/AspNetCore/FunctionsHttpProxyingMiddlewareTests.cs

```diff
@@ -47,18 +51,125 @@ public async Task Middleware_NoOpsOnNonHttpTriggers()
             test.MockCoordinator.Verify(p => p.CompleteFunctionInvocation(It.IsAny<string>()), Times.Never());
         }
 
-        private static (FunctionContext FunctionContext, HttpContext HttpContext, Mock<IHttpCoordinator> MockCoordinator) SetupInputs(string triggerType)
+        [Fact]
+        public async Task SimpleHttpTrigger_ActionResultHandled()
+        {
+            var test = SetupTest("httpTrigger");
+            var mockDelegate = new Mock<FunctionExecutionDelegate>();
+
+            // In a simple HTTP trigger function, there is only one input (the trigger), and the HTTP response
+            // The HTTP response will be in the InvocationResult
+            var mockActionResult = GetMockActionResult();
+            var bindingFeatures = test.FunctionContext.Features.Get<IFunctionBindingsFeature>();
+            bindingFeatures.InvocationResult = mockActionResult.Object;
+
+            var funcMiddleware = new FunctionsHttpProxyingMiddleware(test.MockCoordinator.Object);
+            await funcMiddleware.Invoke(test.FunctionContext, mockDelegate.Object);
+
+            mockActionResult.Verify();
```

Comment:
Not blocking, but it would be good to track improvements to the new tests to improve code reuse/reduce duplication.